### ISSUE TITLE
Alternative name to reduce confusion

### ISSUE THREAD

**Issue by ariya** [Medium/phantomjs#447]
I apologize for reviving an old topic from > 3 years ago (see #10).

After all these years, we (=PhantomJS team) always have the need to explain in many different forms of conversation (Github issue, tweet, email, etc) that this npm-based package is an installer and not the actual PhantomJS itself. We would politely redirect them here to  continue their issues with installation, troubleshooting, availability, or other inquiry. 

Hence why I argued long time ago (in #10) that this confusion could be reduced by giving this project a different name, e.g. `phantomjs-installer`, instead of just the vanilla `phantomjs`. Back then, this thought was driven by my gut-feeling. Now we now that this confusion truly exists due to the repeated explanation we need to give.

In short: I think this project is great, giving easy access to Node.js developers to install PhantomJS. With a minor name tweak, I believe this will reduce confusion for everyone.

Thank you for your consideration!

**Comment by bedney** [comment]
I concur with this, especially given the fact that the version numbers between the two can differ, which causes yet more confusion still. Even after using the 'npm PhantomJS installer' for 2+ years, I myself have to be overly precise about which I'm talking about when talking with others: the core PhantomJS or the npm installer.

Having said that, let me also offer my thanks to the team here - many people use this package as an easy way to install PhantomJS.

**Comment by nicks** [comment]
@ariya is `phantomjs-prebuilt` ok by you?

i'm skeptical -- my impression is that the confusion is less about the name, and more about the (incorrect) assumption that phantomjs-team controls and provides support for all phantomjs distribution channels.

but i'm fine with deprecating `phantomjs` and pointing people to `phantomjs-prebuilt`. worst-case it creates a lot of busy-work for people downstream who have to update their package references. We can always undeprecate if there's a revolt.

**Comment by ariya** [comment]
@nicks Your skepticism might be right, I just think that having the exact same name does not cause people to pause and think, and hence their assumption will not be corrected (ever).

I'm fine with any name, pick something that suits you well.

**Comment by bedney** [comment]
Might I suggest `node-phantomjs`? It would be in keeping with how many other npm packages are named.

Feel free to ignore this comment from the peanut gallery and thanks again to both of you guys for your work on these projects! :smile:

**Comment by ariya** [comment]
Here is an interesting data point: since I filed this issue 11 hours, there have been two install problems that this project probably should be aware of:
* https://github.com/ariya/phantomjs/issues/12970#issuecomment-175688330
* https://github.com/ariya/phantomjs/issues/13951

**Comment by nicks** [comment]
i don't think https://github.com/ariya/phantomjs/issues/12970#issuecomment-175688330 is referring to this package. we have never tried to install phantomjs 2.0.0. (i.e., people are complaining to you about lots of npm installer packages, not just this one)

**Comment by ariya** [comment]
@nicks I'm not implying that the report is valid. I just don't have enough information to immediately verify the claim (without going to this project, check that it never deals with 2.0.0, and so on).

**Comment by nicks** [comment]
@ariya We've agreed to change the name. Are you looking for some additional action?

There are many NPM phantomjs installers with many names. Bugs in them get reported to your repo. Bugs in them get reported to our repo. Not sure if that makes you feel any better. Users are crazy!

**Comment by ariya** [comment]
@nicks Thanks for your consideration and feel free to resolve this issue once an npm package with the new name is published.

**Comment by nicks** [comment]
https://www.npmjs.com/package/phantomjs shows a deprecation notice !

long live https://www.npmjs.com/package/phantomjs-prebuilt !!

**Comment by ariya** [comment]
:+1:

**Comment by bedney** [comment]
Thank You :smiley:

**Comment by just-boris** [comment]
As I understand, there is no need to rename package on NPM. 
The issue here that top packages have the same name: https://github.com/search?utf8=✓&q=phantomjs
The same thing with google search results.
People probably are confused and can get to the wrong project.

Rename project in npm doesn't solve that issue at all because nothing changed in Github.

**Comment by jakub-g** [comment]
Agreed with the last comment of @just-boris, probably you could also consider renaming the GitHub repo.

FYI when you rename a repo on GitHub, then GitHub will put a permanent HTTP-URL and git-URL redirections from new name to the old name, so all the remotes bookmarks / links / git remotes people people have will continue to work. So, it's pretty safe to change repo name on GitHub without any unwanted side-effects.
